<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>General</class>
 <widget class="QWidget" name="General">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Acuerdos Reglamentarios N:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Datos Generales del Acta de Audiencia</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lSerie2"/>
   </item>
   <item row="10" column="1">
    <widget class="QTimeEdit" name="tInicio"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lAutos"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lLegajo"/>
   </item>
   <item row="11" column="1">
    <widget class="QTimeEdit" name="tFin"/>
   </item>
   <item row="9" column="1">
    <widget class="QDateEdit" name="dFecha2"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lAcuerdo2"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Hora de inicio:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lAcuerdo1"/>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="dFecha1"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Hora de finalización: </string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>NUEVA REUNIÓN:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Acuerdos Reglamentarios N:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Legajo N°:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lSerie"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lEtapa"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Serie:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Serie:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="lNueva"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Etapa:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_12213">
     <property name="text">
      <string>Autos:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lLegajo</tabstop>
  <tabstop>lAutos</tabstop>
  <tabstop>lEtapa</tabstop>
  <tabstop>lAcuerdo1</tabstop>
  <tabstop>lSerie</tabstop>
  <tabstop>dFecha1</tabstop>
  <tabstop>lAcuerdo2</tabstop>
  <tabstop>lSerie2</tabstop>
  <tabstop>dFecha2</tabstop>
  <tabstop>tInicio</tabstop>
  <tabstop>tFin</tabstop>
  <tabstop>lNueva</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
